package.json

```diff
@@ -105,13 +105,13 @@
   },
   "dependencies": {
     "ajv": "^5.5.2",
+    "javascriptstuff-db": "^1.12.0",
```

Comment:
This package author deprecated `tool-list`?

Reply:
Current tool-list package fetch data from [this branch](https://github.com/ahfarmer/javascriptstuff-db/tree/v0).
So, at the moment [this](https://webpack.js.org/starter-kits) instructions not work properly because you refer to a file present in master branch.

Reply:
I see the page and links work too?
What is not working?

Reply:
According with [this documentation](https://webpack.js.org/starter-kits) I need to submit a PR against this file:data/source/react-starter-projects.js. (master branch)
But the `tool-list` package fetch data from [another branch](https://github.com/ahfarmer/javascriptstuff-db/tree/v0) (branch v0).
So, there are 2 options:
- use `tool-list` package and update the description in webpack starters kit guide.
- use `javascriptstuff-db` package that fetch data from master branch. 

Note that the repo's list present in master branch is more updated than the other one (branch V0)

Reply:
Ok, I guess it is ok then.